nexus-writer/src/hdf5_handlers/dataset.rs

```diff
@@ -13,10 +15,12 @@ impl HasAttributesExt for Dataset {
         Ok(attr)
     }
 
+    /// This should be a provided method.
```

Comment:
Functions in trait `impl`s should not be documented (they inherit the documentation from the trait).